torch/csrc/jit/script/init.cpp

```diff
@@ -266,6 +296,101 @@ FunctionSchema getSchemaWithNameAndDefaults(
       schema.is_varret());
 }
 
+static Decl computeOverloadDecl(
```

Comment:
it's not just merging default arguments, it's also merging extra parameters from the implementation def

Reply:
the alternative is `mergeDefaultsAndExtraParametersIntoOverload` , which is a little too long imo...

Reply:
In general I think it's better to have a long name than a non-descriptive one. Because if it's non-descriptive, then the user has to read the entire function to find out what it does, instead of just having to parse a long name.